content/concepts/consensus-network/consensus.md

```diff
@@ -161,12 +161,24 @@ Best practices for applications submitting transactions include:
         - Take care to use a server with a continuous ledger history to detect this case <a href="#footnote_10" id="from_footnote_10"><sup>10</sup></a>.
     - It may be necessary to check the status of a transaction repeatedly until the ledger identified by `LastLedgerSequence` is validated.
 
-## Further Resources
+## See Also
+
+- **Concepts:**
+    - [Introduction to Consensus](intro-to-consensus.html)
+    - [Consensus Research](consensus-research.html)
+    - [Ripple Consensus Video](https://www.youtube.com/watch?v=pj1QVb1vlC0)
+- **Tutorials:**
+    - [Reliable Transaction Submission](reliable-transaction-submission.html)
+    - [Run `rippled` as a Validator](run-rippled-as-a-validator.html)
+- **References:**
+    - [Ledger Format Reference](ledger-data-formats.html)
+    - [Transaction Format Reference](transaction-formats.html)
+    - [consensus_info method][]
+    - [validator_list_sites method][]
+    - [validators method][]
+    - [consensus_info method][]
+
 
-- [Consensus White Paper](https://ripple.com/files/ripple_consensus_whitepaper.pdf)
-- [Ledger Format Reference](ledger-data-formats.html)
-- [Ripple Consensus Video](https://www.youtube.com/watch?v=pj1QVb1vlC0)
-- [Reliable Transaction Submission](reliable-transaction-submission.html)
 
 
 
```

Comment:
Nit and out of scope for this PR: we use "End Notes" here and "Footnotes" in "Introduction to Consensus." We should probably use one or the other. I know there's a difference in some contexts but this doesn't seem to be one of them.

Reply:
I agree it makes sense to make this consistent.

I think "end notes" is technically be the more precise term, but "footnotes" is close enough and far more common. (The quantity of Google search results is ~10x in favor of "footnotes" over "endnotes" and "end notes" combined.)

We use "footnotes" in two places and "end notes" just in the one, so I'm changing it to "Footnotes" here too.